lib/index.js

```diff
@@ -367,7 +424,7 @@ exports.init = function (sequelize, optionsArg) {
               var d = RevisionChange.build({
                 path: difference.path[0],
                 document: document,
-                //revisionId: data.id,
+                revisionId: revision.id,
```

Comment:
Do we need to set this here? I think that the relationship is already covered through the associations.

Reply:
I think I did this for a reason because I did not get the reference to properly work

Reply:
I'll have a check and let you know